src/main/java/shortestpath/Transport.java

```diff
@@ -241,21 +232,59 @@ private static List<Quest> findQuests(String questNamesCombined) {
         return quests;
     }
 
-    private static void addItemTransports(Map<WorldPoint, List<Transport>> transports) {
+    private static void addFairyRingTransports(Map<WorldPoint, List<Transport>> transports, String path) {
```

Comment:
The item transports can be parse the same way as the other tsv files so it doesn't need its own function. The fairy rings tsv is handled different, so this function is the same logic from the old `addTransports()` function just split out. It also does the map parsing instead of the column assumption way.